get-involved/discord.html

```diff
@@ -15,8 +15,8 @@
  		<header class="block__heading">
  			<h1 class="block__heading__title">MyBB Discord Server</h1>
  		</header>
- 		<p>MyBB Staff are seperated in the channel by a special username colour, and are probably around to talk to as well as other MyBB users.</p>
- 		<p style="font-weight:bold;">You <span style="text-decoration: underline;">must not</span> disclose vulnerabilities in MyBB within the Discord Server. Instead, please see the <a href="{{ site.baseurl }}/get-involved/security/">Security Research</a> page for information on how to responsibly disclose vulnerabilities.</p>
+ 		<p>MyBB Staff are seperated in the server by a special username colour, and are probably around to talk to as well as other MyBB users.</p>
```

Comment:
While we're fixing things ... MyBB uses US English, colour should be color:

```suggestion
 		<p>MyBB Staff are seperated in the server by a special username color, and are probably around to talk to as well as other MyBB users.</p>
```